keyboards/sawnsprojects/okayu/info.json

```
@@ -0,0 +1,308 @@
+{
+    "keyboard_name": "Okayu",
+    "manufacturer": "Salane",
+    "url": "",
+    "maintainer": "Mai The San",
+    "usb": {
+        "vid": "0x5350",
+        "pid": "0x00A1",
+        "device_version": "1.0.0",
+        "force_nkro": true
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": true,
+        "command": true,
```

Unless there's a specific reason to have these enabled.
```suggestion
        "console": false,
        "command": false,
```